README.md

```diff
@@ -80,13 +80,19 @@ $ sudo apt-get install libc6:i386 libncurses5:i386 libstdc++6:i386
 
 ### Installing mbed CLI
 
+#### Windows Installer
+The [mbed CLI for Windows Installer](https://mbed-media.mbed.com/filer_public/7f/46/7f46e205-52f5-48e2-be64-8f30d52f6d75/mbed_installer_v041.exe) will install all pre-requisites required and works on windows 7 and above. More details on the instaler can be found [here](https://docs.mbed.com/docs/mbed-os-handbook/en/latest/dev_tools/cli_install/)
```

Comment:
IMPORTANT NOTE: The last link will not work once we delete this doc from the YAML because that page won't exist anymore.

- Please lower case "installer" because it is a common noun.
- Please change "will install" to "installs" for consistent tense.
- Please delete the hyphen from "prerequisites".
- Please change "windows 7 and above" to "Windows 7 and later" for capitalization of proper nouns and style.
- Please change the last sentence to "You can find more details about the installer [here]..." for active voice and correct spelling.